.github/actions/multi-approvers/tests/multi-approvers.test.ts

```diff
@@ -0,0 +1,593 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import assert from "node:assert/strict";
+import { test } from "node:test";
+import nock from "nock";
+import {
+  MultiApproversAction,
+  MultiApproversParams,
+} from "../src/multi-approvers";
+
+const GITHUB_API_BASE_URL = "https://api.github.com";
+
+const BASE_PARAMS = {
+  eventName: "pull_request",
+  runId: 1,
+  branch: "twig",
+  pullNumber: 12,
+  repoName: "speed-trap",
+  repoOwner: "acme-org",
+  token: "fake-token",
+  team: "roadrunners",
+  octokitOptions: { request: fetch },
+  // eslint-disable-next-line no-unused-vars
+  logDebug: (_: string) => {},
+  // eslint-disable-next-line no-unused-vars
+  logInfo: (_: string) => {},
+  // eslint-disable-next-line no-unused-vars
+  logNotice: (_: string) => {},
+} as MultiApproversParams;
+
+async function assertDoesNotReject(
+  nockScope: any,
+  paramOverrides: Partial<MultiApproversParams> = {},
+) {
+  const params = Object.assign({}, BASE_PARAMS, paramOverrides);
+  const multiApproversAction = new MultiApproversAction(params);
+  await assert.doesNotReject(() => multiApproversAction.validate());
+  assert(
+    nockScope.isDone(),
+    `Unexecuted nock HTTP mocks: ${nockScope.pendingMocks()}`,
+  );
+}
+
+async function assertRejects(
+  nockScope: any,
+  message: string,
+  paramOverrides: Partial<MultiApproversParams> = {},
+) {
+  const params = Object.assign({}, BASE_PARAMS, paramOverrides);
+  const multiApproversAction = new MultiApproversAction(params);
+  await assert.rejects(() => multiApproversAction.validate(), {
+    name: "Error",
+    message,
+  });
+  assert(
+    nockScope.isDone(),
+    `Unexecuted nock HTTP mocks: ${nockScope.pendingMocks()}`,
+  );
+}
```

Comment:
1. What are all these constants for? Per the TOTT hanging in the bathrooms right now (and my own personal experience) you want to inline test data as much as possible.

2. These helpers names are way too generic. In fact, they conflict with names in the stdlib.